src/lib/models/models.ml

```diff
@@ -0,0 +1,625 @@
+(******************************************************************************)
```

Comment:
Don't know if we can simply this file:
- a lot of local refs
- maybe choose a single output format (smt2) to ease maintenance.